sail/blueprints/sample.yaml

```diff
@@ -0,0 +1,71 @@
+---
+version: 1
+
+# Install plugins from the WordPress.org repo or custom sources.
+plugins:
+  woocommerce: latest
+  wordpress-seo: latest
+  hello-dolly:
+    url: https://downloads.wordpress.org/plugin/hello-dolly.1.7.2.zip
+
+# Install themes from the WordPress.org directory or custom sources.
+themes:
```

Comment:
[feat] theme activation option on deploy

Reply:
Last theme in the list is activated by default :)